### ISSUE TITLE
Google Play Movies

### ISSUE THREAD

**Issue by richardfrost** [richardfrost/AdvancedProfanityFilter#204]
#### :movie_camera: The Site
play.google.com

#### :speech_balloon: Subtitle Element/Node
_Example:_
```html
<div class="lava-tt-window-default lava-timed-text-window lava-tt-window-appearance-2 lava-tt-window-layout-35" style="">
  <div>
    <span class="lava-tt-caption-default lava-timed-text-caption lava-tt-text-format-0">
      Hey, I'm as surprised
      <br>
      as you are.
    </span>
  </div>
  <div style="display: inline-block; position: absolute; visibility: hidden; height: 1em;"></div>
<div class="lava-timed-text-window-width"></div></div>
```

#### :pencil: Additional notes
_Requested on the Chrome Web Store support page_

**Comment by richardfrost** [comment]
I have only had a few minutes to test it, but this config seems to work:

```json
{
  "play.google.com": {
    "className": "lava-timed-text-window",
    "mode": "element",
    "subtitleSelector": "span.lava-timed-text-caption",
    "tagName": "DIV"
  }
}
```

To use the config above, open the filter's Option page, click on the Audio tab, and add it to the "Custom Audio Sites" box on the right side of the page. Click "Save" and refresh/re-open the Google Play page. Also remember that captions/subtitles must be turned on for the muting to work. You can hide them using the options available in the audio tab mentioned above.

_**TIP:** To test the muting capability, identify a word that doesn't need to be filtered and temporarily add it to the list of words to filter._

If anyone gets around to testing this, please let me know how it works for you. From the couple videos I tested it on, it always muted. The caption timing wasn't the greatest on a couple of the videos, but that can happen anywhere.

Also worth noting, it looks like most (if not all) of the videos I can access through Google Play Movies are also available through YouTube under [Purchases](https://www.youtube.com/feed/purchases).

**Comment by richardfrost** [comment]
This should now be included in version [2.7.5](https://github.com/richardfrost/AdvancedProfanityFilter/releases/tag/2.7.5)
